Telegram/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "node-test",
+  "version": "0.0.1",
+  "description": "A server that connects Google's Dialogflow CX API",
```

Comment:
nit: maybe consider to rephrase it as `A server that integrates Dialogflow CX with Telegram`?